store/dataStore.ts

```diff
@@ -0,0 +1,17 @@
+import { create } from 'zustand';
+
+interface Example {
+    id: number;
+    name: string;
+}
+interface ExampleStore {
+    examples: Example[];
+}
+
+const dataStore = () => ({
+    examples: [],
+})
+
+const useUserStore = create<ExampleStore>(dataStore)
```

Comment:
Almost there. This is supposed to be use than what ever variable you name the store. In this case it should be `useDataStore`.